tests/test_embedding.py

```diff
@@ -174,6 +179,34 @@ def test_embed_content_called_with_request_options(self):
 
         self.client.embed_content.assert_called_once_with(request, **request_options)
 
+    @parameterized.named_parameters(
```

Comment:
Why copy this test to here? For most other thinhgs we just add them to the parameterization in the main test.

Reply:
I missed updating async's batch_embedding path with the new param. Thought it might be helpful to catch such mistakes.

I should remove the parametrized. I Will update it soon.

Thanks.

Reply:
Should we add a main test for async code match rather than having it for separate modules?